### ISSUE TITLE
Suggestion: real-debrid integration

### ISSUE THREAD

**Issue by blamezippy** [xbapps/xbvr#1460]
put.io is currently the only available cloud storage.

Having real-debrid available as cloud storage would be great.

real-denrid files can be accessed via their API. WebDAV is also an option but I don't know if there's a performance impact in using that.

**Comment by andrewsnewton** [comment]
Yeah I want the real debrid integration. That's the only cloud storage I use. Please look into it.
